src/resolvers/UserResolver.ts

```diff
@@ -14,17 +22,35 @@ class UserResolver {
     return user;
   }
 
-  @Mutation((_returns) => User)
+  @Mutation((_returns) => User, { nullable: true, description: 'Lets the user from a given <uuid> follow the <Publication> given by <pubID>' })
   async followPublication(@Arg('uuid') uuid: string, @Arg('pubID') pubID: string) {
     const user = await UserRepo.followPublication(uuid, pubID);
     return user;
   }
 
-  @Mutation((_returns) => User)
+  @Mutation((_returns) => User, { nullable: true, description: 'Lets the user from a given <uuid> unfollow the <Publication> given by <pubID>' })
   async unfollowPublication(@Arg('uuid') uuid: string, @Arg('pubID') pubID: string) {
     const user = await UserRepo.unfollowPublication(uuid, pubID);
     return user;
   }
+
+  @Mutation((_returns) => User, { nullable: true, description: 'Adds the <Article> given by the <articleID> to the <User\'s> read articles'})
+  async readArticle(@Arg('uuid') uuid: string, @Arg('articleID') articleID: string) {
+    const user = await UserRepo.appendReadArticle(uuid, articleID);
+    return user;
+  }
+
+  @Mutation((_returns) => User, { nullable: true, description: 'Increments the number of bookmarks for the <User> given by <uuid>'  })
+  async bookmarkArticle(@Arg('uuid') uuid: string) {
+    const user = await UserRepo.incrementBookmarks(uuid);
+    return user;
+  }
+
+  @Mutation((_returns) => [User], {description: 'Creates and returns the Weekly Debrief for all users'})
```

Comment:
'Creates and returns...' but in the other create functions we did not talk about returning. Stay consistent with the documentation so that there are no ambiguities (oh this one creates, but does it return?)